### ISSUE TITLE
Cannot add property timeout, object is not extensible

### ISSUE THREAD

**Issue by peterver** [proxycrawl/proxycrawl-node#2]
I'm getting the following error `Cannot add property timeout, object is not extensible` when importing the CrawlingAPI.

Environment: Node v 10.22.0-alpine

**Comment by peterver** [comment]
Nevermind, the object that I was passing with the token was frozen with Object.freeze. Could be interesting to change:

```
this.options = options;
```

into 

```
this.options = Object.assign({}, options);
```

on the following line to prevent this from happening to other people -> https://github.com/proxycrawl/proxycrawl-node/blob/7becd841043b2d9083316be54e8bf438b3aa2b93/src/base-api.js#L26

**Comment by peterver** [comment]
I've opened a PR #3 to fix this :)

**Comment by crawlbase** [comment]
Thank you for the PR. This is now released in version 2.1.4
